trusted_shuffler/trusted_shuffler/src/tests.rs

```diff
@@ -27,23 +30,29 @@ impl RequestHandler for TestRequestHandler {
     // A test function that processes requests.
     // In the real use-case, this function should send requests to the backend and
     // return responses.
-    async fn handle(
-        &self,
-        request: TrustedShufflerRequest,
-    ) -> anyhow::Result<TrustedShufflerResponse> {
+    async fn handle(&self, request: PlaintextRequest) -> anyhow::Result<PlaintextResponse> {
         if drop_request(&request) {
             // In that case we don't send an answer ever.
             loop {
                 tokio::task::yield_now().await;
             }
         }
-        Ok(generate_response(&request))
+        Ok(generate_plain_response(&request))
     }
 }
 
 // Non-async version of the `handle_request` function used to create expected responses.
-fn generate_response(request: &TrustedShufflerRequest) -> TrustedShufflerResponse {
-    TrustedShufflerResponse {
+fn generate_plain_response(request: &PlaintextRequest) -> PlaintextResponse {
```

Comment:
Generating would mean that the Trusted Shuffler generates something by its own. But it just proxies requests.
I think it could be renamed to `get_plaintext_response` or something similar

Reply:
Done. Thanks.